components/automate-chef-io/content/docs/iam-v2-overview.md

```diff
@@ -152,23 +151,102 @@ The default policies are: Viewers, Editors, Administrator, and Ingest.
 
 Just add members to any of these default policies, as described later, to quickly get your basic users up and running.
 
-### Policies and Roles in Automate
+## Working with Projects
 
-<!-- I do not like saying "user interface" because technically that includes the command line, too. But I can live with it if you think that is the lesser of two evils. -->
-IAM v2 introduces partial policy management from within Chef Automate in your browser.
+IAM projects are collections of resources created in Chef Automate or ingested from external data providers, including ChefInfra and ChefInspec.
+Projects are used in a policy to reduce the scope of permissions of that policy to just the resources tagged with the given projects.
+Without a policy statement to allow an operation tagged with a given project, a user cannot perform the actions of that policy.
+Note that Chef Automate is currently limited to six projects while we continue to refine the user experience during this beta period.
+
+There are four steps to setting up projects:
+
+1. Define each project via the UI (or the command line if you prefer).
+   Start on the projects list page shown previously to create your projects.
+2. Create or modify IAM policy statements to restrict permissions to specific projects as needed.
+   *Every* statement must either indicate specific projects or specify the wildcard denoting all projects (`*`).
+   By default, any previously existing policies are automatically setup with that wildcard so they apply to all projects.
+3. Tag resources with projects (see [Tagging Resources]({{< relref "iam-v2-overview.md#tagging-resources" >}})).
+4. Select the projects to filter in the UI.
+   Once you create some projects, use the **global project filter** in the penultimate position of the top navigation bar to select one or more projects for viewing.
+   With no selection (the default), you are thereby saying you want *no* filtering, hence displaying resources for *all* projects for which you have permission.
```

Comment:
```suggestion
   With no selection (the default), all resources you have permission on will be displayed.
```

Reply:
i dont think we need to explain what _no selection_ means in terms of filtering, instead just that no selection means you see all resources you have access to.. i think the addition of content makes it more difficult to understand